pkg/apis/serving/v1/revision_lifecycle.go

```diff
@@ -171,6 +171,18 @@ func (rs *RevisionStatus) PropagateDeploymentStatus(original *appsv1.DeploymentS
 func (rs *RevisionStatus) PropagateAutoscalerStatus(ps *autoscalingv1alpha1.PodAutoscalerStatus) {
 	// Reflect the PA status in our own.
 	cond := ps.GetCondition(autoscalingv1alpha1.PodAutoscalerConditionReady)
+	if ps.ActualScale != nil {
+		rs.ActualReplicas = *ps.ActualScale
+	} else {
+		rs.ActualReplicas = 0
```

Comment:
I think this else may be redundant since ActualReplicas defaults to 0 anyway. Might be clearer to drop it.

Reply:
I think it's fine to guard against the `PodAutoscalerStatus` and leave this else clause

ie. if in the future PodAutoscalerStatus scale could cycle through the values -1 => some number > 0 => -1 then we'd be fine.

Or alternatively it might be better to have

```go
rs.ActualReplicas := Math.Max(ps.GetActualScale(),0)
```

Reply:
Doesn't work, since Math.Max works with float64 only (thanks golang). But we can write our own max (we have a few in the code already).